src/components/DataTableController.spec.js

```diff
@@ -36,14 +36,54 @@ describe('DataTableController', () => {
         { name: 'Maude', age: 48 }
       ];
 
-      setController({ 
+      setController({
         options: options,
         rows: rows
       });
 
       ctrl.$onInit();
     });
 
+    it('should have assign defaults', () => {
+      let expectedOptions = {
+        checkboxSelection: false,
+        columnMode: 'standard',
+        columns: [
+          { prop: 'name', sort: 'asc' },
+          { prop: 'age' }
+        ],
+        emptyMessage: 'No data to display',
+        footerHeight: 0,
+        headerHeight: 30,
+        internal: {
+          offsetX: 0,
+          offsetY: 0,
+          innerWidth: 0,
+          bodyHeight: 300,
+        },
+        loadingIndicator: false,
+        loadingMessage: 'Loading...',
+        multiSelect: false,
+        paging: {
+          count: 0,
+          offset: 0,
+          mode: null,
+          size: 10,
+        },
+        reorderable: true,
+        rowHeight: 30,
+        scrollbarV: true,
+        selectable: false,
+        sortable: true,
+        sortType: 'multiple',
+      };
+
+      let options = ctrl.options;
+      delete options.$outer;      
```

Comment:
Oh right, you can't use angular.copy on an object that contains Scope instances. I don't think this should cause problems anyway.